main.go

```diff
@@ -182,6 +182,14 @@ func init() {
 		RunE:  contextualize(handler.Up, handler.Panic),
 	})
 
+	logsCmd := &cobra.Command{
+		Use:   "logs",
+		Short: "View the most-recent deploy's logs",
+		RunE:  contextualize(handler.Logs, handler.Panic),
+	}
+	logsCmd.Flags().Int32P("num_lines", "n", 0, "--lines")
```

Comment:
I think you have the args a bit confused

```suggestion
	logsCmd.Flags().Int32P("lines", "n", 0, "output a specific number of lines")
```